### PR TITLE
Brush up NPC Skewer automation

### PR DESCRIPTION
**Pull request by simonward**

### CODE REVIEW

File: packs/pfs-season-2-bestiary/perizia-5-6.json [hunk 1]
```diff
@@ -1452,7 +1452,24 @@
...
-                "rules": [],
+                "rules": [
+                    {
+                        "domain": "damage-roll",
+                        "key": "RollOption",
+                        "option": "skewer",
+                        "toggleable": true
+                    },
+                    {
+                        "damageType": "bleed",
```
Comment (simonward) [inline packs/pfs-season-2-bestiary/perizia-5-6.json:1463]: Not sure on how strict PFS is on critical bleed being 2d4 vs (2*1d4).

File: packs/pathfinder-bestiary/drow-fighter.json [hunk 2]
```diff
@@ -1397,13 +1397,14 @@
                         "value": 1
                     },
                     {
-                        "key": "Note",
+                        "damageType": "bleed",
```
Comment (simonward) [inline packs/pathfinder-bestiary/drow-fighter.json:1400]: Should this double on a crit?

Comment (TikaelSol) [reply packs/pathfinder-bestiary/drow-fighter.json:1400]: yes

File: packs/pfs-season-2-bestiary/perizia-5-6.json (hunk 1 above)
Comment (TikaelSol) [reply packs/pfs-season-2-bestiary/perizia-5-6.json:1463]: The system should be using whatever setting the GM has chosen.

**Pull request merged**